src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -89,6 +89,51 @@ private class ACCT_IndividualAccounts_TEST {
         system.assertEquals(null,insertedContacts[0].AccountId);
     }
     
+    /*********************************************************************************************************
+    * @description Accounts created from the trigger should populate the currency code with the contact 
+    */
+    @IsTest
+    private static void accountCreatedWithCorrectCurrencyCodes() {
+        //Skip the test if Advancement is installed
+        if (ADV_PackageInfo_SVC.useAdv()) return;
+
+        Boolean isMultiCurrencyEnabled = UserInfo.isMultiCurrencyOrganization();
+        String defaultCurrency = UserInfo.getDefaultCurrency();
+
+        Contact newContact = new Contact(
+            LastName = 'CurrencyTest',
+            FirstName = 'Test'
+        );
+        
+        String currencyField = '';
+        if (isMultiCurrencyEnabled) {
+
+             // Set currency to be something different than the default one if possible
+            List<SObject> currencies = Database.query('SELECT IsoCode FROM CurrencyType WHERE IsActive = True');
```

Comment:
How about using `UTIL_UnitTestData_TEST.nonDefaultCurrencyType` for this test? See similar tests in `PMT_Payment_TEST`.